recipes/3seq/meta.yaml

```diff
@@ -31,3 +31,6 @@ about:
   license: CC BY-NC-SA 4.0
   license_file: LICENSE.md
   summary: '3SEQ tests all sequence triplets in an alignment for a mosaic recombination signal.'
+
```

Comment:
This most probably needs bump of `build > number` and adding `run_exports`.
IMO dependencies should add support for `linux-aarch64` in their own PRs.